payments-core/src/main/java/com/stripe/android/StripeIntentResult.kt

```diff
@@ -86,13 +85,4 @@ abstract class StripeIntentResult<out T : StripeIntent> internal constructor(
             const val TIMEDOUT: Int = 4
         }
     }
-
-    private companion object {
-        private val PROCESSING_IS_SUCCESS = setOf(
```

Comment:
nit: I think it's useful to leave a set with verbose names instead of inline the items in `#hasDelayedSettlement`, could we keep the set and have `hasDelayedSettlement` check against the set??

Reply:
I think `PaymentMethod.Type` is the right location to keep this list. What do you mean by verbose, just `PaymentMethod.Type.SepaDebit` instead of `SepaDebit`? That's redundant when inside `PaymentMethod.Type`.

Reply:
oh by verbose I meant still save the types inside a set with a name, and check against that set. I guess it's fine to move the checks into the Type class iteself